ftw/contentpage/tests/test_event_views.py

```diff
@@ -100,6 +101,12 @@ def setUp(self):
                                        endDate=DateTime(2013, 05, 30, 11, 00),
                                        location="Bern")
 
+        self.eventfolder.invokeFactory('EventPage', 'event7', title="Event7",
+                                       wholeDay=True,
+                                       startDate=DateTime(2013, 05, 20, 4, 0),
+                                       endDate=DateTime(2013, 05, 20, 5, 0),
```

Comment:
I don't like this huge test setup.
I'd maybe break local consistency and move the event folder creation to the test.
This would fix the currently failing other tests..